models/member.ini

```diff
@@ -10,6 +10,7 @@ type = string
 [fields.level]
 label = Level
 type = select
+;source = site.query('/contributing/membership')
 choices = platinum, gold, silver, bronze, adhoc, professional, individual, inkind
 choice_labels = Platinum, Gold, Silver, Bronze, Professional, Individual, Ad hoc, In kind
```

Comment:
Ok, to try to unify, what is the difference between in kind, ad hoc, and other.

This is in order to unify a bit the different options